VimeoUpload/VimeoUploader.swift

```diff
@@ -27,7 +27,7 @@
 import Foundation
 import VimeoNetworking
 
-public class VimeoUploader<T: VideoDescriptor>
+open class VimeoUploader<T: VideoDescriptor>
```

Comment:
Right now there are two classes in the example projects that subclass `VimeoUploader`: `NewVimeoUploader` and `OldVimeoUploader`. Should we change the scope of `VimeoUploader` to `public` then?